ATv2: distinguish acc act/inact events (#8202)
nrfconnect/sdk-nrf: applications/asset_tracker_v2/doc/sensor_module.rst

Both events contain an accelerometer threshold value ``accelerometer_threshold`` in m/s2, present in the event structure.

> Motion detection in the module is enabled and disabled according to the mode setting of the device.

Reviewer: ```suggestion
Motion detection is enabled and disabled according to the device mode parameter, received in the aforementioned configuration events.
```

> Motion detection is enabled and disabled according to the device mode parameter, received in the aforementioned configuration events.
> Motion detection will be enabled in passive mode and disabled in active mode.

Reviewer 2: ```suggestion
Motion detection is enabled and disabled according to the device mode parameter, received in the configuration events.
It is enabled in the passive mode and disabled in the active mode.
```

The sensor module will send out a :c:enum:`SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED` event if it detects movement.
Similarly, :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` is sent out if there was no movement for a while.

> 

Reviewer: Remember to update the `sensor_module.rst` docs in https://github.com/nrfconnect/sdk-nrf/pull/8020 after this is merged to include:
- Documentation on how activity/inactivity thresholds as well as inactivity time is set. Both compile time and real-time. The initial section in `Motion detection` can probably be expanded to encapsulate more configs.
- Refer to the inactivity threshold instead of `for a while` so that users know exactly what parameter the should manipulate to extend the inactivity time.

> The sensor module will send out a :c:enum:`SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED` event if it detects movement.
> Similarly, :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` is sent out if there was no movement for a while.

Reviewer 2: ```suggestion
The sensor module sends out a :c:enum:`SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED` event if it detects movement.
Similarly, :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` is sent out if there is no movement in a while.
```
Should we be more specific with the time? How long is a while?

Author: This will be made more specific in an upcoming PR when we enable users to configure this at runtime.

> This functionality acts as flow control.

Author: Like this? ```Motion detection in the module is enabled and disabled according to the mode setting of the device.
Motion detection will be enabled in passive mode and disabled in active mode.```
